src/triage/component/postmodeling/utils/config_file_example.yaml

```diff
@@ -0,0 +1,16 @@
+# The relevant experiment hash
+experiment_hash: 'hash_of_the_experiment'
+
+# Path where the models and matrices are stored
+project_path: 'path/to/models/and/matrices'
+
+# Model group ids we need predictions for
+# List of integers
+model_group_ids:
+  - 1
+  - 2
+
+# This is optional. 
+# If not specificied, all the models in the group will be scored
+# Should be a list of two items
```

Comment:
Should probably specify whether these end points will be included in the range.

Reply:
yep, added.